File: back/yourpilling/src/main/java/com/ssafy/yourpilling/analysis/model/service/AnalysisService.java
package com.ssafy.yourpilling.analysis.model.service;

import com.ssafy.yourpilling.analysis.model.service.vo.in.AnalysisVo;
import com.ssafy.yourpilling.analysis.model.service.vo.out.OutAnalysisVo;

public interface AnalysisService {
    OutAnalysisVo analysis(AnalysisVo vo);
}

File: wearable/yourpilling/app/src/main/java/com/ssafy/yourpiliing/presentation/retrofit/analysis/AnalysisService.kt
package com.ssafy.yourpilling.presentation.retrofit.analysis

import kotlinx.coroutines.Deferred
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query

interface AnalysisService {

    @GET("/api/v1/pill/analysis")
    fun analysis() : Call<AnalysisResponse>
}